lang/de.json

```diff
@@ -13,15 +13,15 @@
         "SettingUserinfoUrl": "Userinfo URL",
         "SettingUserinfoUrlHelp": "z.B. https://<USERNAME>.eu.auth0.com/userinfo",
         "SettingUserinfoId": "Userinfo ID",
-        "SettingUserinfoIdHelp": "Name des Feldes, in dem die Benutzer-ID enthalten ist. Normalerweise, für OpenID Connect Dienste wie Auth0, ist das 'sub'. Github gibt die eindeutige Benutzer-ID in dem Feld 'id' an.",
+        "SettingUserinfoIdHelp": "Name des Feldes, in dem die Benutzer-ID enthalten ist. Normalerweise, für OpenID Connect Dienste wie Auth0, Keycloak oder Unikname Connnect, ist das 'sub'. Github gibt die eindeutige Benutzer-ID in dem Feld 'id' an.",
```

Comment:
The help text should be kept short.
Auth0 is the most popular commercial auth service and Keycloak the most popular selfhosted, so they should be enough.

Reply:
Removed